listenbrainz/db/stats.py

```diff
@@ -25,10 +25,9 @@
 import ujson
 
 from listenbrainz import db
-from listenbrainz import config
+from listenbrainz import config 
 
-
-def insert_user_stats(user_id, artists, recordings, releases, artist_count):
+def insert_user_stats(user_id, artists, recordings, releases, artist_count, timestamp):
     """Inserts user stats calculated from Google BigQuery into the database.
```

Comment:
is BigQuery true here now? This docstring should probably be updated to say Spark, or to be technology independent.

Reply:
Yes. I think it should be replaced with 'Spark' because that is the technology we are using.